1.10/deploying-services/containerizers/docker-containerizer.md

```diff
@@ -29,5 +38,10 @@ To specify the Docker containerizer from the DC/OS CLI, add the following parame
 }
 ```
 
-- [Follow a Docker app tutorial](/docs/1.10/deploying-services/creating-services/deploy-docker-app/).
+**Important:** If you leave the `args` field empty, the default entry point will be the launch command for the container. If your container does not have a default entry point, you must specify a command in the `args` field. If you do not, your service will fail to deploy.
+
```

Comment:
this is likely incorrect / unintended.   I don't think it is the `args` field that you mean.    In addition there is no args field in the UI.    It is the command field.   If a command isn't provided the default entry point is used... if there is not default entry point then you need to specify a `command` not an arg.

more detail here: https://mesosphere.github.io/marathon/docs/native-docker.html
under Commands vs Args